charts/ingress-nginx/templates/admission-webhooks/validating-webhook.yaml

```diff
@@ -12,10 +12,10 @@ webhooks:
   - name: validate.nginx.ingress.kubernetes.io
     rules:
       - apiGroups:
-          - extensions
```

Comment:
@aledbf what's rationale to drop validating `Ingress`es with `extensions/v1beta1`? `networking.k8s.io` is available since k8s 1.14 but while extensions is supported why not keep validating those too? Ingress `extensions/v1beta1` API will be dropped only in v1.22 (see https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)

Reply:
@sslavic because the client-go v1.19 removed the extensions package.
If you have ingresses in the old `extensions/v1beta1` will be converted to `networking.k8s.io`, the version used in ingress-nginx,  by the API server

Reply:
> but while extensions is supported why not keep validating those too?

Because `networking.k8s.io/v1` only works in v1.19 and we need to support 1.16, 1.17, 1.18, and 1.19
(not sure we can keep doing this long term)